template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/s3guard/S3ConfigProvider.java

```diff
@@ -35,9 +40,17 @@
 
     private static final String S3GUARD_TABLE_TAG_VALUE = "s3guard";
 
+    private static final String S3_BUCKET_ENDPOINT_PARAM_TEMPLATE = "fs.s3a.bucket.%s.endpoint";
+
+    private static final String S3_ENDPOINT_TEMPLATE = "s3.%s.amazonaws.com";
```

Comment:
Have not checked this, just asking out of curiosity: Is the endpoint DNS name always formed this way? Does it always exist for every possible region?

Reply:
The doc gives the general format of regional endpoints: https://docs.aws.amazon.com/general/latest/gr/rande.html#regional-endpoints. It does, however, not state if it works for all regions.
Specifically for s3 endpoints I scrolled through the s3 endpoints and all (except for the two Chinese regions) seemed to comply: https://docs.aws.amazon.com/general/latest/gr/s3.html